internal/commands/user.go

```diff
@@ -83,12 +81,15 @@ func getUsersInConversation(
 ) (*[]model.User, error) {
 	var users []model.User
 
+	logWriter := app.Logger.With(
+		log.NewValue("channelID", channelID),
+	)
+
 	usersIDs, err := getUsersIDsInConversation(ctx, app, channelID)
 	if err != nil {
-		app.Logger.Error(
+		logWriter.Error(
 			ctx,
 			"command/user.getUsersInConversation",
```

Comment:
Add a log.Action("GetUsersDIsInConversation") to identify in what part of the func the error was thrown.